tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "moduleResolution": "node",
     "importHelpers": true,
     "resolveJsonModule": true,
-    "target": "es2015",
+    "target": "es2017",
     "module": "es2020",
     "lib": ["es2018", "dom"],
```

Comment:
new standard for new browser projects
- except, for libraries, the target remains es2015